docs/coding-guidelines/libraries-packaging.md

```diff
@@ -56,17 +56,6 @@ Microsoft.Extensions.Logging.Abstractions.NullLogger</PackageDescription>
 
 Package content can be defined using any of the publicly defined Pack inputs: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets
 
-### TargetFrameworks
```

Comment:
Does it make sense to document somewhere that targeting a .NETFramework TFM automatically adds a RID-specific asset when the package contains other RID-specific assets which would apply?

Reply:
Sure, will do.